lib/common/errors.ts

```diff
@@ -159,7 +160,7 @@ export class Errors implements IErrors {
 		} catch (ex) {
 			const loggerLevel: string = $injector.resolve("logger").getLevel().toUpperCase();
 			const printCallStack = this.printCallStack || loggerLevel === "TRACE" || loggerLevel === "DEBUG";
-			const message = printCallStack ? resolveCallStack(ex) : `\x1B[31;1m${ex.message}\x1B[0m`;
+			const message = printCallStack ? resolveCallStack(ex) : isInteractive() ? `\x1B[31;1m${ex.message}\x1B[0m` : ex.message;
```

Comment:
It seems this probably will affect Sidekick/KinveyStudio but I'm not sure what exactly will be the impact.

Reply:
It might be better to split the conditions in two separate lines for readability of the code.